### ISSUE TITLE
Cards with Evolution can't be recognized properly

### ISSUE THREAD

**Issue by capt0083** [kosciukiewicz/sbc-solver#8]
I have two evolved cards which are showing as original cards rating in the solver.

**Comment by galoniz** [comment]
not only that - it fails to categorize them properly in general.

**Comment by kosciukiewicz** [comment]
That's true. That is planned to be added. Please note, for now my priority is to integrate the solver with https://futmind.com/ with cooperation with it's creator that will open new opportunities like optimizing the prices. Then we will try to add another missing features to the solver itself.
